scrunch/datasets.py

```diff
@@ -252,6 +252,7 @@ class Project:
     def __init__(self, project_resource):
         self.resource = project_resource
         self.url = self.resource.self
+        self._order = ProjectDatasetsOrder(self.resource)
```

Comment:
Why is the project resource needed to init the order? I think that the init args here should be:
 * self.resource.datasets <- the datasets catalog
 * self.resource.datasets.order <- the order of the catalog we want this class to represent (because there may be more than one order)